.github/workflows/test-ci.yml

```diff
@@ -54,12 +54,18 @@ jobs:
 
             -   name: Install Dependencies
                 run: yarn
+                env:
+                    YARN_IGNORE_NODE: 1
```

Comment:
Can't we set the env at the workflow level? https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow makes it look like we can.

Reply:
I was too lazy to google this (as I initially thought I needed it only in one place), thanks!